dissect/target/loaders/zip.py

```diff
@@ -0,0 +1,97 @@
+import logging
```

Comment:
Could you add tests for this new functionality? This could be done later, if you decide to go on the `AcquireLoader` route.